lib/tx_build/default.ex

```diff
@@ -13,20 +13,21 @@ defmodule Stellar.TxBuild do
     TransactionEnvelope
   }
 
-  @type signature :: Signature.t()
-  @type signatures :: signature() | list(signature())
-  @type operation :: Operation.t()
-  @type operations :: operation() | list(operation())
+  @behaviour Stellar.TxBuild.Spec
+
+  @type tx :: Transaction.t()
+  @type signatures :: Signature.t() | list(Signature.t())
+  @type tx_envelope :: TransactionEnvelope.t() | nil
 
   @type t :: %__MODULE__{
-          tx: Transaction.t(),
+          tx: tx(),
           signatures: signatures(),
-          tx_envelope: TransactionEnvelope.t() | nil
+          tx_envelope: tx_envelope()
         }
 
   defstruct [:tx, :signatures, :tx_envelope]
```

Comment:
@luishurtado I'm not really sure where this structure should be on. `Stellar.TxBuild.Default` or `Stellar.TxBuild`?

What do you think?

Reply:
`Stellar.TxBuild` since `Stellar.TxBuild.Default` is "just" an implementation.

Reply:
Got it! Thanks!